CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+cmake_minimum_required(VERSION 3.13.0)
+project(hss1374 VERSION 1.0.0)
+set(CMAKE_PROJECT_HOMEPAGE_URL "https://www.mixxx.org")
+set(CMAKE_PROJECT_DESCRIPTION "High-speed MIDI-over-Firewire device access library for Windows and Mac OSX.")
+
+#
+# Build
+#
+
+add_library(hss1394
+  code/src/1394API.cpp
+  code/src/ConfigRom.cpp
+  code/src/HSS1394.cpp
+)
+target_include_directories(hss1394 PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/code/src")
+
+if(APPLE)
+  target_sources(hss1394 PRIVATE code/src/osx/1394API_OSX.cpp)
+
+  find_library(IOKit_LIBRARY IOKit REQUIRED)
+  target_link_libraries(hss1394 PUBLIC ${IOKit_LIBRARY})
+
+  find_library(CoreFoundation_LIBRARY CoreFoundation REQUIRED)
+  target_link_libraries(hss1394 PUBLIC ${CoreFoundation_LIBRARY})
+
+  find_library(Carbon_LIBRARY IOKit REQUIRED)
+  target_link_libraries(hss1394 PUBLIC ${Carbon_LIBRARY})
+
+  if(CMAKE_OSX_DEPLOYMENT_TARGET STREQUAL "10.4u")
+    target_compile_definitions(hss1394 PRIVATE "OSX10_4")
+  endif()
```

Comment:
I think we can get rid of whatever hack required this.

Reply:
No idea if macOS 10.4 is still a thing. Doesn't hurt to leave it in for now.

Reply:
It's not a thing anymore. macOS 10.4 was released in 2005. For reference, the 10 year old Macbook Air I've been borrowing is on macOS 10.13 because Apple won't let it upgrade to any newer OS.